### PR TITLE
feat: add tools dropdown

### PR DESCRIPTION
**Pull request by RavenInd**
- tools dropdown

### CODE REVIEW

File: docs/src/components/ui/DesktopMenu.tsx [hunk 1]
```diff
@@ -134,12 +135,161 @@ const SocialLinksItem = styled.li`
...
+`;
+
+const MenuItemDropdown = styled.div`
+  --bg-color: ${tm(({ colors }) => colors.neutral0)};
+  width: 494px;
+  height: 176px;
+  box-shadow: 0px 9px 28px 8px rgba(0, 0, 0, 0.05);
+  filter: drop-shadow(0px 6px 50px rgba(10, 11, 13, 0.41));
+  border-radius: 4px;
+  background-color: var(--bg-color);
```
Comment (usulpro) [inline docs/src/components/ui/DesktopMenu.tsx]: Why can't we set bg color directly as we did it everywhere before? If the reason that we need this in the child element - can't we use `inherit` value instead (see a comment below)

File: docs/src/components/ui/DesktopMenu.tsx [hunk 2]
```diff
@@ -134,12 +135,161 @@ const SocialLinksItem = styled.li`
...
+    z-index: -1;
+    position: absolute;
+    top: -6px;
+    left: 50%;
+    transform: translateX(-50%) rotate(45deg);
+    transform-origin: center;
+    content: " ";
+    width: 16px;
+    height: 16px;
+    background-color: var(--bg-color);
```
Comment (usulpro) [inline docs/src/components/ui/DesktopMenu.tsx]: wouldn't inherit works?

```suggest
background-color: inherit;
```

**Review by usulpro — APPROVED** [review]
Please check my comments and feel free to merge as you updated the styles

File: docs/src/components/ui/DesktopMenu.tsx (hunk 2 above)
Comment (RavenInd) [reply docs/src/components/ui/DesktopMenu.tsx]: Corrected

File: docs/src/components/ui/DesktopMenu.tsx (hunk 1 above)
Comment (RavenInd) [reply docs/src/components/ui/DesktopMenu.tsx]: Corrected

**Pull request merged**